### PR TITLE
Add weekly update 2016-02-22

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

File: locale/en/blog/weekly-updates/weekly-update.2016-02-22.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### Notable changes
+
+* **buffer**
+  * make byteLength work with Buffer correctly (Jackson Tian)
+    - [#4738](https://github.com/nodejs/node/pull/4738)
+* **debugger** 
+  * guard against call from non-node context (Ben Noordhuis)
+    - [#4328](https://github.com/nodejs/node/pull/4328)
+    - fixes segfaults in debugger
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-02-22.md]: This should not be in the separate line, right?

**Comment by evanlucas** [comment]
Maybe make mention of the new logo?

**Comment by JungMinu** [comment]
Thanks, updated :)

**Comment by stevemao** [comment]
What about the release of v5.7.0?

File: locale/en/blog/weekly-updates/weekly-update.2016-02-22.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - [#4738](https://github.com/nodejs/node/pull/4738)
+* **debugger** 
+  * fixes segfaults in debugger and guard against call from non-node context (Ben Noordhuis)
+    - [#4328](https://github.com/nodejs/node/pull/4328)
+  * do not incept debug context (Myles Borins)
+    - [#4819](https://github.com/nodejs/node/pull/4819)
+    - fixes crash in debugger when using util methods
+   
+### New official Node.js logo
+As a result of an iteration on the official logo, We are proud to introduce and unveil out new logo<br>
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-02-22.md]: we not We. Also our not out. And br tag is not needed

File: locale/en/blog/weekly-updates/weekly-update.2016-02-22.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+date: 2016-02-22T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-02-22
+layout: blog-post.hbs
+---
+
+### Node.js News
+Node v4.3.1 (LTS) is released.
+
```
Comment (MylesBorins) [inline locale/en/blog/weekly-updates/weekly-update.2016-02-22.md:13]: we should likely also mention the v4.4.0-rc that we really need people to test out

**Comment by Trott** [comment]
With all these last-minute comments, I wonder if these would get more visibility if the issue to gather items for these things would be opened in this repo rather than (or in addition to?) the evangelism repo. 

Nothing other than these weekly updates seems to be going on in the evangelism repo lately anyway, so interested parties can probably try watching that repo too.

**Comment by Trott** [comment]
Maybe for the v5.7.0 item, link to the relevant blog post? http://nodejs.org/en/blog/release/v5.7.0/

**Comment by JungMinu** [comment]
Thanks, revised :)

**Comment by JungMinu** [comment]
@Trott I will gather items for these things in this repo :smile:

**Comment by JungMinu** [comment]
All nits addressed and introduced Node v4.4.0 (RC) :heart:

**Pull request merged**